stixcore/tmtc/packet_factory.py

```diff
@@ -115,7 +116,10 @@ class ValidationFunctionError(AttributeError):
     """
 
 
-# Main packet class
-idbm = IDBManager(Path(__file__).parent.parent / "data" / "idb")
-GenericPacket.idb_manager = idbm
+if __name__ == "__main__":
```

Comment:
Will this code every be called as main?

Reply:
separated it into test and prod code blocks